pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/CaseTransformFunctionTest.java

```diff
@@ -116,6 +145,90 @@ public void testCaseTransformFunctionWithIntResults() {
     }
   }
 
+  @Test
+  public void testCaseTransformFunctionWithoutCastForFloatValues() throws Exception {
+    Map<String, DataSource> dataSourceMap = new HashMap<>();
+    int numRows = 1;
+    final int[] intSVValues = new int[numRows];
```

Comment:
(minor, convention) we don't usually put `final` for local variable

Reply:
resolved